test/karma.conf.js

```diff
@@ -44,35 +46,22 @@ module.exports = function(config) {
       'bower_components/zeroclipboard/dist/ZeroClipboard.min.js',
       'bower_components/angular-mocks/angular-mocks.js',
       // endbower
-      // application dependencies
-      'app/common/constants.js',
-      'app/common/application/init.js',
-      'app/common/application/services/applicationService.js',
-      'app/common/ui-helper/init.js',
-      'app/common/ui-helper/spinner.js',
-      'app/common/auth/init.js',
-      'app/common/auth/user.js',
-      'app/common/auth/userService.js',
-      'app/common/auth/authentication.js',
-      'app/common/domain/init.js',
-      'app/common/domain/services/locationService.js',
-      'app/common/domain/services/configurationService.js',
-      'app/common/config/init.js',
-      'app/common/config/services/configurations.js',
-      'app/common/app-framework/init.js',
-      'app/common/app-framework/models/appDescriptor.js',
-      'app/common/app-framework/services/appService.js',
-      // application files to test
-      'app/home/init.js',
-      'app/home/initialization.js',
-      'app/home/app.js',
-      'app/home/controllers/*.js',
+      // bundled libraries
+      'lib/modernizr.custom.80690.js',
+      // application files
+      'app/**/*.js',
```

Comment:
We are now loading the full application into the browser so that we can get our true test coverage metric.